elastic/apm-server
# model: change Host.IP to an array

## Motivation/summary

ECS states that `host.ip` should hold an array of values. Let's align with ECS before we settle on a stable encoding.

## Checklist

~- [ ] Update [package changelog.yml](https://github.com/elastic/apm-server/blob/master/apmpackage/apm/changelog.yml) (only if changes to `apmpackage` have been made)~

## How to test these changes

1. Run apm-server, send events
2. Check a transaction sample's metadata, `host.ip` should be rendered sensibly in the flyout in the UI
3. Check the document's _source, `host.ip` should be an array (even if there's just one IP).

## Related issues

https://github.com/elastic/apm-server/issues/4120
